pkg/apis/operator/v1alpha1/tektonpipeline_defaults.go

```diff
@@ -80,6 +80,20 @@ func (p *Pipeline) setDefaults() {
 		p.MetricsTaskrunLevel = config.DefaultTaskrunLevel
 	}
 
+	// Resolvers
```

Comment:
are we enabling by default?

Reply:
yes

Reply:
yes starting in 0.41 they are enabled by default.